android/app/build.gradle

```diff
@@ -151,6 +152,9 @@ android {
 }
 
 dependencies {
+    // Multidex support for < Android 5.0 (API level 21)
```

Comment:
looks like this PR also includes the Multidex PR #116?

Reply:
Yeah sorry, didn't split it out properly. I'll rebase and remove this